test/SuperResponsiveTable.test.js

```diff
@@ -1,12 +1,25 @@
 import React from 'react'
 import { render } from '@testing-library/react'
 import '@testing-library/jest-dom/extend-expect'
-import { Table, Thead, Tbody, Tr, Th, Td } from '../src/SuperResponsiveTable'
+import * as baseStyleUtils from '../src/baseStyleUtils'
+import {
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  TBaseStyles,
+} from '../src/SuperResponsiveTable'
 
 describe('SuperResponsiveTable', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
   // START OF COMPONENT SETUP
-  const setup = overrides => {
-    const { getAllByTestId, getByTestId, getAllByText } = render(
+  const setup = (ui, options) => {
```

Comment:
Make `setup` more flexible (follow [the React Testing Library `render` API](https://testing-library.com/docs/react-testing-library/api#render)) to allow it to be used in a wider range of tests.

Maybe we can [refactor this into a custom render](https://testing-library.com/docs/react-testing-library/setup#custom-render) in the future?